apps/api/src/shared/service/user-state.service.ts

```diff
@@ -20,8 +21,25 @@ export class UserStateService {
   async findByUser(userId: string): Promise<UserState> {
     const state: UserState = await this.userStateModel.findOne({ user: userId }).exec();
     if (state) {
-      state.logos = await this.userCompletedLogoService.findByState(state._id);
+      // state.logos = await this.userCompletedLogoService.findByState(state._id);
     }
     return state;
   }
+
+  async verifyValidatedLogo(logoId: string, userId: string): Promise<boolean> {
+    const logos = await this.getUserLogos(userId);
+    console.log(logos);
```

Comment:
do we need this?

Reply:
Remoted all console.logs, thanks for noticing